data/json/items/fluff.json

```
@@ -361,5 +361,19 @@
     "color": "light_cyan",
     "intelligence": 5,
     "time": "10 m"
+  },
+  {
+    "type": "GENERIC",
+    "id": "train_data_card",
+    "symbol": ",",
+    "color": "white",
+    "name": { "str_sp": "laboratory shipping records" },
+    "description": "Some type of advanced data storage device, filled with logistical data on subterranean train routes and schedules.",
+    "price": 10000,
+    "price_postapoc": 50,
+    "material": [ "plastic" ],
+    "weight": "18 g",
```

You've got a test failing because of the density of this item.   It's currently set at over 9 times the density of plastic based on the ml.
```suggestion
    "weight": "1 g",
```